mozci/console/commands/push.py

```diff
@@ -165,14 +167,117 @@ def handle(self):
                 )
 
 
+class ClassifyEvalCommand(Command):
+    """
+    Evaluate the classification results for a given push (or a range of pushes) by comparing them with reality.
+
+    classify-eval
+        {branch=mozilla-central : Branch the push belongs to (e.g autoland, try, etc).}
+        {--rev= : Head revision of the push.}
+        {--from-date= : Lower bound of the push range (as a date in yyyy-mm-dd format).}
+        {--to-date= : Upper bound of the push range (as a date in yyyy-mm-dd format).}
+        {--medium-confidence=0.8 : If recalculate parameter is set, medium confidence threshold used to classify the regressions.}
```

Comment:
We should print an error if one of these parameters is passed but `recalculate` is not